src/components/Roadmap.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import data from 'data.json';
+
+//styles
+import '../index.scss';
+
+//types
+import { Status, STATUSES } from 'types';
+
+export const Roadmap = () => {
+  let plannedCount = data.productRequests.filter((item) => {
```

Comment:
Great use of filter 👍

Do you know the difference between `const` and `let`? Which one is appropriate to use in these cases?

Reply:
Const can't be reassigned a new value, so that's why I used let since the value of planned content changes. But now that I'm thinking about it, the value isn't getting reassigned. So I think I would just use const instead.

Reply:
You got it!